src/nvim/tui/tui.c

```diff
@@ -229,16 +230,20 @@ void tui_handle_term_mode(TUIData *tui, TermMode mode, TermModeState state)
   bool is_set = false;
   switch (state) {
   case kTermModeNotRecognized:
-  case kTermModePermanentlySet:
   case kTermModePermanentlyReset:
+    // TODO(bfredl): This is really ILOG but we want it in all builds.
+    // add to show_verbose_terminfo() without being too racy ????
```

Comment:
Off-topic: should we add a terminal feature section to `:checkhealth`? (@gpanders)